types/jasmine/index.d.ts

```diff
@@ -146,7 +145,7 @@ declare namespace jasmine {
     function createSpy(name?: string, originalFn?: Function): Spy;
 
     function createSpyObj(baseName: string, methodNames: SpyObjMethodNames): any;
-    function createSpyObj<T>(baseName: string, methodNames: SpyObjMethodNames): SpyObj<T>;
+    function createSpyObj<T>(baseName: string, methodNames: SpyObjMethodNames<T>): SpyObj<T>;
 
     function createSpyObj(methodNames: SpyObjMethodNames): any;
     function createSpyObj<T>(methodNames: SpyObjMethodNames): SpyObj<T>;
```

Comment:
Should the same change be applied here (i.e. `methodNames: SpyObjMethodNames<T>`)?